MAVEN.md

```diff
@@ -0,0 +1,34 @@
+#Publishing to maven.
```

Comment:
Put an space between `#` and `P`.
Otherwise Github don't render it correctly:
![bildschirmfoto 2018-06-14 um 16 29 38](https://user-images.githubusercontent.com/10229883/41418506-272dec6a-6ff0-11e8-9d40-9631a5175021.png)